scripts/util.py

```diff
@@ -175,16 +136,14 @@ def add_vocab_csvs(data_dir: Path, nlp, append: bool = True):
     files = [str(f) for f in files]
     
     for file in files:
-        try:
-            wb_type = file.split('wikidata_')[1].split('s.csv')[0].upper()
-            water_bodies[wb_type] = []
-            df = pd.read_csv(file)
-            for i in range(len(df)):
-                if (type(df['Name'][i]) is str and
-                        len(df['Name'][i]) > 1 and
-                        (not re.search('^Q[0-9]+', df['Name'][i]))):
-                    water_bodies[wb_type].append((df['Name'][i].lower(), df['ID'][i]))
-        except Exception as e:
-            raise RuntimeError(f'Could not load {file}.\nError: {str(e)}')
-    
-    add_vocab(water_bodies, nlp, append=append)
\ No newline at end of file
+        wb_type = file.split('wikidata_')[1].split('s.csv')[0].upper()
+        water_bodies[wb_type] = []
+        df = pd.read_csv(file)
+        for i in range(len(df)):
+            if (type(df['Name'][i]) is str and
+            len(df['Name'][i]) > 1 and
+            (not re.search('^Q[0-9]+', df['Name'][i]))):
+                names = name_split(df['Name'][i])
+                # split 'amazon river' or 'river amazon' to 'amazon'
+                for name in names: water_bodies[wb_type].append((name, df['ID'][i]))
```

Comment:
nit pick: no one liners please. best to keep formatting consistent

Reply:
Made the change.